pandas/tests/io/pytables/test_store.py

```diff
@@ -650,7 +650,9 @@ def test_coordinates(setup_path):
 
         expected = concat([df1, df2], axis=1)
         expected = expected[(expected.A > 0) & (expected.B > 0)]
-        tm.assert_frame_equal(result, expected)
+        tm.assert_frame_equal(result, expected, check_freq=False)
+        # FIXME: 2021-01-18 on some (mostly windows) builds we get freq=None
```

Comment:
can you open an issue about this so we don't forget. could be an actual bug ( as it recently started appearing)

Reply:
#39197 

> as it recently started appearing

im pretty sure i semi-recently tightened this, was too aggressive in doing so